autosubmit/job/job_list.py

```diff
@@ -1377,14 +1370,17 @@ def _manage_job_dependencies(self, dic_jobs, job, date_list, member_list, chunk_
             if dic_jobs.as_conf.jobs_data.get(aux_key, {}).get("RUNNING", "once") == "chunk":
                 if aux_key in distances_of_current_section and distance > distances_of_current_section[aux_key]:
                     distances_of_current_section[aux_key] = distance
-            elif dic_jobs.as_conf.jobs_data.get(aux_key, {}).get("RUNNING", "once") == "member":
-                if aux_key in distances_of_current_section_member and distance > distances_of_current_section_member[aux_key]:
-                    distances_of_current_section_member[aux_key] = distance
+
+            elif dic_jobs.as_conf.jobs_data.get(aux_key, { }).get("RUNNING", "once") == "member":
```

Comment:
No need to have a blank space between `{}`. This actually adds a warning in PyCharm, 

![image](https://github.com/user-attachments/assets/945dd59e-840c-4435-b8a0-50c419f36b5d)